addons/hr_contract/models/hr_employee.py

```diff
@@ -132,6 +132,10 @@ def _get_calendar(self, date_from=None):
             return res
         return contracts[0].resource_calendar_id.sudo(False)
 
+    def _get_current_calendar(self):
```

Comment:
There's already the ``_get_calendar`` method serving that purpose, which is better suited as it takes in parameter the target date, instead of taking the current one.

Remove that method and use ``_get_calendar`` instead.